### PR TITLE
Adding a simple mechanism to configue and create BT Connections.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import com.google.api.client.repackaged.com.google.common.base.Preconditions;
+
+/**
+ * This class provides a simplified mechansim of creating a programmatic Bigtable Connection.
+ */
+public class BigtableConfiguration {
+  private static final String[] CONNECTION_CLASS_NAMES = {
+    "com.google.cloud.bigtable.hbase1_0.BigtableConnection",
+    "com.google.cloud.bigtable.hbase1_1.BigtableConnection",
```
Comment (carterpage) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: Are we prioritizing 1_0 over 1_1? (getConnectionClass returns the first one in order which matches)

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  };
+
+  private static Class<? extends Connection> CONNECTION_CLASS = getConnectionClass();
```
Comment (carterpage) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: final?

**Comment by carterpage** [comment]
Couple of minor questions. PTAQL.

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java (hunk 1 above)
Comment (sduskis) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: The implementations are mutually exclusive.  HBase 1.0 and 1.1 can't coexist in the same VM.

**Comment by carterpage** [comment]
LGTM

**Pull request merged**